### PR TITLE
Add a script to update etcd VM images

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: cloud/google/update_etcd_vm_image.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# TODO(alcutter): Factor out common code with update_mirror_vm_image.sh script
```
Comment (Martin2112) [inline cloud/google/update_etcd_vm_image.sh]: Not sure if thats > 80 chars or just how GH is displaying it.

Comment (Martin2112) [inline cloud/google/update_etcd_vm_image.sh]: This script just recreates what's there? Might be worth a comment on how safe it is to run.

Comment (AlCutter) [inline cloud/google/update_etcd_vm_image.sh]: 79 chars I think

Comment (AlCutter) [inline cloud/google/update_etcd_vm_image.sh]: I've added a bunch more comments about what the script does.

**Pull request merged**